partcad/src/partcad/shape.py

```
@@ -135,57 +209,51 @@ def do_change(self, change=None):
-
-    def show(self, show_object=None):
-        asyncio.run(self.show_async(show_object))
-
-    def shape_info(self):
-        asyncio.run(self.get_wrapped())
+            components = []
+            # TODO(clairbee): consider removing this exception handler permanently
+            # Comment out the below exception handler for easier troubleshooting in CLI
+            try:
+                components = await self.get_components(ctx)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)